clippy_lints/src/lib.rs

```diff
@@ -226,8 +222,19 @@ pub fn register_plugins(reg: &mut rustc_plugin::Registry<'_>) {
                     .emit();
             toml::from_str("").expect("we never error on empty config files")
         }
-    };
+    }
+}
 
+pub fn register_pre_expansion_lints(session: &rustc::session::Session, store: &mut rustc::lint::LintStore, conf: &Conf) {
```

Comment:
can you move this function above the `read_conf` function to reduce the diff size?